src/hardware/sblaster.cpp

```diff
@@ -2858,21 +2865,25 @@ bool SB_GetAddress(uint16_t &sbaddr, uint8_t &sbirq, uint8_t &sbdma)
 
 static void sblaster_mixer_callback([[maybe_unused]]const int requested_frames)
 {
+	assert(sblaster && sblaster->channel);
```

Comment:
Minor: I prefer separate asserts for such scenarios, but it's borderline 😄

Reply:
Definitely agree; more obvious when an assert trips. 
Switched them all.